db/db_properties_test.cc

```diff
@@ -2106,6 +2106,114 @@ TEST_F(DBPropertiesTest, GetMapPropertyBlockCacheEntryStats) {
   ASSERT_EQ(3 * kNumCacheEntryRoles + 4, values.size());
 }
 
+TEST_F(DBPropertiesTest, GetMapPropertyWriteStallStats) {
```

Comment:
TODO: adding a seperate UT to verify format of writes stall cause/condition/internal stats enum and hardcoded map in write_stall_stats.cc